site/sfguides/src/getting_started_with_document_ai/getting_started_with_document_ai.md

```diff
@@ -0,0 +1,570 @@
+                if save:
+                     with st.spinner("Saving document approval..."):
+                        for i in range(approve_checkboxes):
+                            if not st.session_state[f"check_{i}"]:
+                                st.error("Need to apporve all checks before saving")
+                                st.stop()
+                        # Create a SQL to save that the document is verified
+                        insert_sql = f"INSERT INTO {doc_ai_context}.{doc_ai_source_verify_table} (file_name, verification_date) VALUES ('{sel_doc}', '{datetime.now().isoformat()}')"
+                        _ = session.sql(insert_sql).collect()
+                        st.success("✅ Success!")
+                        # Rerun is used to force the application to run from the begining so we can not verify the same document twice
+                        st.experimental_rerun()
+            # Display of PDF section
+            with col_doc:
+                if 'pdf_page' not in st.session_state:
+                    st.session_state['pdf_page'] = 0
+    
+                if 'pdf_url' not in st.session_state:
+                    st.session_state['pdf_url'] = sel_doc
+                
+                if 'pdf_doc' not in st.session_state or st.session_state['pdf_url'] != sel_doc:
+                    pdf_stream = session.file.get_stream(f"@{doc_ai_context}.{doc_ai_doc_stage}/{sel_doc}")
+                    pdf = pdfium.PdfDocument(pdf_stream)
+                    st.session_state['pdf_doc'] = pdf
+                    st.session_state['pdf_url'] = sel_doc
+                    st.session_state['pdf_page'] = 0
+                
+                nav_col1, nav_col2, nav_col3 = st.columns(3)
+                with nav_col1:
+                    if st.button("⏮️ Previous", on_click=previous_pdf_page):
+                        pass    
+                with nav_col2:
+                    st.write(f"page {st.session_state['pdf_page'] +1} of {len(st.session_state['pdf_doc'])} pages")
+                with nav_col3:
+                    if st.button("Next ⏭️", on_click=next_pdf_page):
+                        pass
+                        
+                display_pdf_page()
+
+```  
+![Streamlit code](assets/streamlit_code.png)  
+5. Open the **Packages** menu  
+![Streamlit package menu](assets/streamlit_packages_menu.png)  
+6. Enter **pypdfium2** in **Find Packages**, and select the first result  
+![Streamlit package search](assets/streamlit_package_search.png)  
+7. Verify that **pypdfium2** is now in the list of **Installed Packages**  
+![Streamlit installed packages](assets/streamlit_installed_packages.png)  
+8. Click **Run** to see the result of the code, in order to hide the code you can click on the **Close editor** icon in the left bottom.  
+![Streamlit close editor](assets/streamlit_close_editor_icon.png)  
+
+You can now start verifying documents.
+<!-- ------------------------ -->
+## Conclusion
```

Comment:
@sfc-gh-mstellwall Pl make this step similar to https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html#8 by renaming the label and added Resources section.